battleship.py

```diff
@@ -62,7 +62,13 @@ def mousePressed(data, event, board):
 Returns: 2D list of ints
 '''
 def emptyGrid(rows, cols):
-    return
+    grid=[]
+    for i in range(rows):
+        l1=[]
+        for j in range(cols):
+            l1.append(EMPTY_UNCLICKED)
+        grid.append(l1)
+    return grid
 

```

Comment:
Maintain a proper naming conversions.

Reply:
I have added a comment for l1 in the file but its not getting updated in GitHub